src/RecoveryServices/RecoveryServices.SiteRecovery/ReplicationProtectedItem/NewAzureRmRecoveryServicesAsrReplicationProtectedItem.cs

```diff
@@ -491,6 +491,17 @@ public class NewAzureRmRecoveryServicesAsrReplicationProtectedItem : SiteRecover
         [Parameter(ParameterSetName = ASRParameterSets.HyperVSiteToAzure)]
         public string UseManagedDisk { get; set; }
 
+        /// <summary>
+        ///     Gets or sets a value indicating whether managed disks should be used during replication.
+        /// </summary>
+        [Parameter]
+        [ValidateNotNullOrEmpty]
+        [ValidateSet(
+            Constants.True,
+            Constants.False)]
+        [Parameter(ParameterSetName = ASRParameterSets.HyperVSiteToAzure)]
+        public string UseManagedDisksForReplication { get; set; }
```

Comment:
Hi, I suggest to use switch parameter as the type of UseManagedDisksForReplication. For switch parameter, if it presents,    that means true. otherwise, it means false

Reply:
We have another parameter named UseMangedDisks which is also defined similarly. To be in consistent with it, I have added UseManagedDisksForReplication similar to that.

Reply:
Switch parameter is our best practice, see https://github.com/Azure/azure-powershell/blob/main/documentation/development-docs/design-guidelines/parameter-best-practices.md#bool-vs-switchparameter. Both options look good to me.